freshmaker/handlers/koji/rebuild_flatpak_application_on_module_ready.py

```diff
@@ -217,8 +221,14 @@ def _filter_images_with_higher_rpm_nvr(self, images):
             private_key=conf.lightblue_private_key,
             event_id=self.current_db_event_id,
         )
-        images = lb.get_images_by_nvrs(images, rpm_nvrs=errata_rpm_nvrs)
-        return images
+
+        container_images = (
+            lb.get_images_by_nvrs(images, rpm_nvrs=errata_rpm_nvrs)
+            if errata_rpm_nvrs
+            else lb.get_images_by_nvrs(images)
+        )
```

Comment:
This is a bit confusing syntax.

```suggestion
        if errata_rpm_nvrs:
            return lb.get_images_by_nvrs(images, rpm_nvrs=errata_rpm_nvrs)

        return lb.get_images_by_nvrs(images)
```

Reply:
Ok. Updated